Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -1308,7 +1322,7 @@ void OnClientSceneInternal(NetworkConnection conn, SceneMessage msg)
         /// <para>Unity calls this on the Server when a Client connects to the Server. Use an override to tell the NetworkManager what to do when a client connects to the server.</para>
         /// </summary>
         /// <param name="conn">Connection from client.</param>
-        public virtual void OnServerConnect(NetworkConnection conn) {}
+        public virtual void OnServerConnect(NetworkConnection conn) { }
```

Comment:
could you disable this rule in your IDE? :)

Reply:
There is no setting for that in Visual Studio - it will always put a space in there.

Reply:
can you unstage the changes at least? :)

Reply:
done